core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -247,7 +247,6 @@ angular.module('oppia').directive('questionsList', [
               ctrl.getGroupedSkillSummaries().current.length;
             var allSkillSummaries = sortedSkillSummaries.map(
               function(summary) {
-                summary.isSelected = false;
```

Comment:
Have you tested this in the topic editor, when trying to create questions?

Reply:
yup, i did that. Also you can see `questions-list-select-skill-and-difficulty-modal.controller.ts` to see the changes I did.

Reply:
Also while testing that I found a bug #10120 and I fixed that. (it was only one line fix)